sample-compose/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 /** Use a separate property for the sample so we can test with different versions easily. */
-val sampleComposeVersion = Versions.Compose
+val sampleComposeVersion = "1.0.0-alpha03"
```

Comment:
could also extract this as "Versions.MinSupportedCompose" ?

Reply:
Sure, although it would be `MaxSupportedCompose`. Maybe we should extract min/max versions and run both in CI?

Reply:
#100